esti/ops/docker-compose.yaml

```diff
@@ -25,7 +25,7 @@ services:
   postgres:
     image: "postgres:11"
     ports:
-      - "5432:5432"
+      - "5433:5432"
```

Comment:
Why changing the default?

Reply:
My intention was to run this environment with as less disturbance to the user as possible. i.e. if he already has a postgres instance running on default port for his private environment, I don't want to make the user take it down just to run the tests

Reply:
Gotcha. Still think it's more intuitive to have the default port. Non-blocking..

Reply:
It should be non consequential for 99.9% of the users, and at least from my experience :) there's more chance there will be a conflict with the default port rather than it being an issue because it's defined on a different port